packages/clay-management-toolbar/src/ManagementToolbar.tsx

```diff
@@ -33,7 +34,9 @@ const ClayManagementToolbar: React.FunctionComponent<IProps> & {
 			}
 		)}
 	>
-		<div className="container-fluid container-fluid-max-xl">{children}</div>
+		<ClayLayout.ContainerFluid size="xl">
```

Comment:
Is `size` really necessary as [it's the default](https://github.com/liferay/clay/blob/master/packages/clay-layout/src/ContainerFluid.tsx#L20) already?

Reply:
it is not necessary... i just forgot there was a default